docs/ballerina-by-example/examples/array-of-arrays/array-of-arrays.bal

```diff
@@ -1,25 +1,25 @@
 import ballerina/io;
 
 function main (string[] args) {
-    // This is how you can initialize an array of int arrays.
+    // This is how you can initialize an array of integer arrays.
```

Comment:
can we just say 'int' instead of integer? 'int' has a specific meaning.

Reply:
Let's go with int array.